spec/factories/file_set.rb

```diff
@@ -27,5 +27,13 @@
     factory :geo_vector_file_set do
       file_metadata FileMetadata.new(mime_type: ControlledVocabulary.for(:geo_vector_format).all.first.value, use: Valkyrie::Vocab::PCDMUse.OriginalFile)
     end
+
+    factory :audio_file_set do
+      file_metadata [
+        FileMetadata.new(mime_type: "audio/wav", use: Valkyrie::Vocab::PCDMUse.OriginalFile, id: "original"),
```

Comment:
the av ingests are coming in without `OriginalFile`; they have `PreservationMaster` instead.